plans/all.fmf

```diff
@@ -0,0 +1,46 @@
+adjust:
+    when: initiator is not defined or initiator != packit
+    discover+:
+        dist-git-source: true
+
+discover+:
+    how: fmf
+execute:
+    how: tmt
+
+/basic_check:
+    summary: Run basic checks
+    discover+:
+        test: /test/basic_check
```

Comment:
How about using `tag` filter instead of name ones? than you can just pick up specific tests from tags on upstream and donwstream

Reply:
I added the `upstream and downstream` tag filters in the latest version. Seems to work and simplifies all.fmf.

It's hard for me to tell, but is it the case that all tests with a particular tag get run sequentially? Or are they still parallelized wherever possible?

Reply:
> It's hard for me to tell, but is it the case that all tests with a particular tag get run sequentially? Or are they still parallelized wherever possible?

tmt still does not have the concept of parallelizing test runs. Can assume it will be sequentially for the foreseeable future.